packages/engine/lib/orchestrator/src/process/local.rs

```diff
@@ -105,13 +105,9 @@ impl process::Command for LocalCommand {
         let child = cmd
             .spawn()
             .wrap_err_lazy(|| format!("Could not run command: {process_path:?}"))?;
-        debug!(
-            "Spawned local engine process for experiment {}",
-            &self.experiment_id
-        );
+        debug!("Spawned local engine process for experiment");
```

Comment:
Why removing the id from the debug output?

Reply:
The ID isn't exactly humanly readable anyway (before the "ID" here was actually the name so it was somewhat readable), and we only have one experiment per CLI run at the moment so it is redundant information.